### DISCUSSION TITLE
Sprint 3 Goal

### DISCUSSION THREAD

**Discussion by dameebsu** [cs481-ekh/f22-kitty-cat-studios#200] [Sprint Planning]
AI Team Sprint Goal: Enhance AI movement and spawning system. Start fixing bugs.

**Comment by annika-mccain** [comment]
General Team Sprint Goal: Finish polishing powerups, sound effects and visual cues for hits, redesign and add functionality to the selection screen, finish the new level (again), and runner testing.
